spec.bs

```diff
@@ -2104,6 +2081,19 @@ An auction config is a [=struct=] with the following items:
 
 </dl>
 
+<div algorithm="validate and convert per buyer signals">

```

Comment:
A `<div algorithm>` with a single `<dfn>` inside it, defaults the value of the `algorithm` attribute: https://speced.github.io/bikeshed/#algorithms

```suggestion
<div algorithm>
```

Reply:
Ack. I probably have a bunch more of these to change.

Reply:
Done.